app/assets/javascripts/angular/components/peopleFiltersPanel/peopleFiltersPanel.component.js

```diff
@@ -83,7 +83,18 @@ function peopleFiltersPanelController(
         };
     };
 
-    this.updateFilters = () => {
+    this.updateFilters = (option, questionId) => {
+        const questionFilter =
+            option === 'no_response' || option === 'any_response' ? option : '';
+
+        if (questionId) {
+            const questions = this.filters.questions;
+            this.filters.questions = {
+                ...questions,
```

Comment:
I don't think you need the `questions` variable.
```suggestion
                ...this.filters.questions,
```

Reply:
yup, true, should have cleaned that up, was attempting another way before this version.